### DISCUSSION TITLE
lazer crashes when changing to or from fullscreen

### DISCUSSION THREAD

**Discussion by Gamingzone123** [ppy/osu#25754] [Q&A]
Hi 
i pressed F11 to change the display mode of lazer but it hung and crashed
it seems to happen when i change in or out of fullscreen mode
here are log files:

[performance-audio.log](https://github.com/ppy/osu/files/13667982/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/13667983/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/13667984/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/13667985/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/13667986/runtime.log)
[updater.log](https://github.com/ppy/osu/files/13667987/updater.log)
[network.log](https://github.com/ppy/osu/files/13667980/network.log)
[performance.log](https://github.com/ppy/osu/files/13667981/performance.log)
[database.log](https://github.com/ppy/osu/files/13667977/database.log)
[input.log](https://github.com/ppy/osu/files/13667978/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/13667979/legacy-ipc.log)

**Comment by bdach** [comment]
There is an error here but I have no idea why it is here.

```
2023-12-14 03:22:33 [error]: An unhandled error has occurred.
2023-12-14 03:22:33 [error]: SharpGen.Runtime.SharpGenException: HRESULT: [0x80070005], Module: [General], ApiCode: [E_ACCESSDENIED/General access denied error], Message: [Access is denied.
2023-12-14 03:22:33 [error]: ]
2023-12-14 03:22:33 [error]: at SharpGen.Runtime.Result.ThrowFailureException()
2023-12-14 03:22:33 [error]: at Vortice.DXGI.IDXGIFactory.CreateSwapChain(IUnknown device, SwapChainDescription desc)
2023-12-14 03:22:33 [error]: at Veldrid.D3D11.D3D11Swapchain.recreateSwapchain()
2023-12-14 03:22:33 [error]: at osu.Framework.Platform.GameHost.DrawFrame()
2023-12-14 03:22:33 [error]: at osu.Framework.Threading.GameThread.processFrame()
```

I would start with trying to update GPU drivers and seeing if it helps.

**Reply by Gamingzone123** [reply]
drivers are most recent version

**Reply by bdach** [reply]
Then I dunno sorry. Workaround for now would probably be to change renderers.

Debugging this further would require a proper trace. If you're willing to try then a tutorial is here: https://nvidia.custhelp.com/app/answers/detail/a_id/3507/~/how-to-share-feedback-for-low-fps%2Fstutter%2Fperformance-issues-using-gpuview. If you decide trying this, **do not post the resulting file publically.** You can e-mail it to my publically-listed github address instead.

**Reply by Gamingzone123** [reply]
ill look into it

**Comment by Gamingzone123** [comment]
@bdach i have emailed you the trace file, please let me know when you have downloaded it as it takes up a fair bit of space on my google drive

**Reply by bdach** [reply]
Yeah these traces are large. They compress pretty well but still large.

I've downloaded the file, thanks. I'll let you know when I have anything to report back. Might be a few days, these are rather difficult to read.
